index.bs

```diff
@@ -184,12 +179,15 @@ Integration with HTML {#html}
 
 This specification integrates with the [[!HTML]] specification by patching the algorithms below:
 
-Document object initialization {#document-init}
+Navigation fetch {#navigation-fetch}
+----------
+
+At [=process a navigate fetch=], after step 13.4 call [$retrieve the Client Hints set$] with <var ignore>currentURL's origin</var> as input.
+
+Navigation response {#navigation-response}
 ----------
 
-At <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#initialise-the-document-object">Create and initialize a Document object</a>,
-after step 11, starting with "Initialize a Document's CSP list",
-call [$initialize the Client Hints set$] with <var ignore>document</var>'s [=relevant settings object=] and |response| as inputs.
+At [=process a navigate response=], after step 7 call [$update the Client Hints set$] with the [=relevant settings object=] and |response| as inputs.
```

Comment:
This moves the CH set initialization to happen a bit higher up the call stack, presumably to cover cases where a document is not currently initialized but where we want Accept-CH to be reflected in the CH set. Can you confirm and outline what those cases are?

Reply:
That's correct, it's to ensure the client hints set is updated on HTTP redirect by processing the response headers for any Accept-CH cache changes. (https://github.com/WICG/client-hints-infrastructure/issues/41)

Reply:
Makes sense. Can you add a comment to the PR's description making this explicit?